Gemfile.lock

```diff
@@ -523,29 +539,29 @@ GEM
       sprockets (>= 3.0.0)
     stackprof (0.2.17)
     statesman (8.0.3)
-    terminal-table (3.0.1)
+    terminal-table (3.0.2)
       unicode-display_width (>= 1.1.1, < 3)
     thor (1.1.0)
     thread (0.2.2)
     thread_safe (0.3.6)
     tilt (2.0.10)
     timecop (0.9.4)
     ttfunk (1.7.0)
-    turbo_tests (1.2.4)
+    turbo_tests (1.3.0)
       bundler
       parallel_tests (~> 3.3)
-      rspec (~> 3.10.0)
-    twilio-ruby (5.54.1)
+      rspec (~> 3.10)
+    twilio-ruby (5.61.2)
       faraday (>= 0.9, < 2.0)
       jwt (>= 1.5, <= 2.5)
       nokogiri (>= 1.6, < 2.0)
     tzinfo (2.0.4)
       concurrent-ruby (~> 1.0)
     unf (0.1.4)
       unf_ext
-    unf_ext (0.0.7.7)
-    unicode-display_width (1.7.0)
-    valid_email2 (3.4.0)
+    unf_ext (0.0.8)
+    unicode-display_width (1.8.0)
+    valid_email2 (4.0.1)
```

Comment:
This change gives us a DNS timeout for our DNS-based email validiation, which is great! https://github.com/lostapathy/valid_email2/commit/ca75ba16723a6e8ac34e6c0e06259cb4c8d58654